<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="user_name">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>User_Name</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>961</width>
      <height>591</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>961</width>
      <height>591</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Олимпиады</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>60</y>
        <width>181</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="add">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;После добавления не забудьте обновить!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;
color: blue;
</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">maingroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;
color: blue;
</string>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download">
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Внимание!!! Загружается только csv файл с разделителем &amp;quot;;&amp;quot;. В файле  6 столбцов(название, предмет обязательно прописать, в остальных, в случае отсутствия, поставить пробел). Строку с заголовками не надо писать!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;afafsd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;
color: blue;
</string>
         </property>
         <property name="text">
          <string>Загрузить</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">maingroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upload">
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;
color: blue;
</string>
         </property>
         <property name="text">
          <string>Выгрузить</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">maingroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="filter_label">
             <property name="styleSheet">
              <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Фильтровать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="filter">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="find_label">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="find_Edit">
           <property name="minimumSize">
            <size>
             <width>451</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
border-radius: 10px;
border: 3px solid rgb(85, 85, 255);
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="olympiad_table"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>450</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: blue;
</string>
      </property>
      <property name="text">
       <string>Не забывайте
обновлять после
каждого добавления!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Результаты</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>280</y>
        <width>250</width>
        <height>272</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>272</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>272</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="filter_label_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Фильтровать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="filter_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="find_label_2">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="find_Edit_2">
           <property name="minimumSize">
            <size>
             <width>440</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>440</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
border-radius: 10px;
border: 3px solid rgb(85, 85, 255);
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="olympiad_table_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="upload_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>750</y>
      <width>271</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 3px solid #5555ff;

</string>
    </property>
    <property name="text">
     <string>Ответы на частые вопросы</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="maingroup"/>
 </buttongroups>
</ui>
